lam4doc/entrypoints/ui/views.py

```diff
@@ -27,23 +30,33 @@ def index():
     return render_template('index.html', title='LAM index page')
```

Comment:
if the title is that hardcoded, then perhaps it is better to place it int the template, rather than the python code.